README.md

```diff
@@ -3,6 +3,7 @@
 [![Latest Version on Packagist][ico-version]][link-packagist]
 [![Software License][ico-license]](LICENSE)
 [![Build Status][ico-travis]][link-travis]
+[![Build status](https://ci.appveyor.com/api/projects/status/github/php-amqplib/php-amqplib?svg=true)](https://ci.appveyor.com/api/projects/status/php-amqplib/php-amqplib)
```

Comment:
Should this be your `ramunasd` AppVeyor instance? We will probably never have one for the `php-amqplib` org since I can't contact the org owners (ATTN: @nubeiro !!!!!R)

Reply:
Well, I don't mind if there is no other options. Maybe we could contact github support regarding ownership transfer?

Reply:
I have already contacted them and they can't help.

Reply:
I removed that badge.